app/views/handlers/section_summary.py

```diff
@@ -1,20 +1,44 @@
-from app.views.handlers.content import Content
+import flask_babel
+from flask import url_for
+
 from app.views.contexts import SectionSummaryContext
 
 
-class SectionSummary(Content):
-    @property
-    def rendered_block(self):
-        return self._render_block(self.block["id"])
+class SectionSummary:
+    def __init__(self, schema, questionnaire_store, current_location, router):
+        self._schema = schema
+        self._questionnaire_store = questionnaire_store
+        self.current_location = current_location
+        self._router = router
+        self._section_id = current_location.section_id
+        self._list_item_id = current_location.list_item_id
+        self._routing_path = router.routing_path(
+            section_id=self._section_id, list_item_id=self._list_item_id
+        )
+        self.page_title = schema.get_title_for_section(self.current_location.section_id)
 
-    def get_context(self):
+    def can_access_section_summary(self):
+        return self._schema.is_summary_in_section(
+            self.current_location.section_id
+        ) and self._questionnaire_store.progress_store.is_section_complete(
+            self._section_id, self._list_item_id
+        )
+
+    def context(self):
         section_summary_context = SectionSummaryContext(
-            self._language,
+            flask_babel.get_locale().language,
```

Comment:
`language` should be passed in, as we do for block handlers so that there isn't a flask_babel dependency here.